test/Mono.Linker.Tests.Cases/Reflection/TypeUsedViaReflectionInvalidName.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Mono.Linker.Tests.Cases.Reflection
+{
+	public class TypeUsedViaReflectionInvalidName
+	{
+		/// <summary>
+		/// This test verifies that if `TypeParser.ParseTypeName` hits an exception and returns null that the linker doesn't fail
+		/// </summary>
+		public static void Main ()
+		{
+			var type = Type.GetType ("System.Collections.Generic.List`1[GenericClass`1[System.String]+Nested]");
```

Comment:
I think it'd be better to add it to existing tests in test/Mono.Linker.Tests.Cases/Reflection/TypeUsedViaReflection.cs

Reply:
Done